src/features/pass-survey/model/AnswerValidator.ts

```diff
@@ -26,69 +51,382 @@ export function AnswerValidator(
   const currentPipelineItem = items?.[step];
   const currentAnswer = answers?.[step];
 
+  const getTimeBasedOnFieldName = (
+    fieldName: string,
+    timeRange: { startTime: HourMinute | null; endTime: HourMinute | null },
+  ): HourMinute | null => {
+    return fieldName === 'from' ? timeRange.startTime : timeRange.endTime;
+  };
```

Comment:
This one isn't being returned with the `AnswerValidator` function. Can be either removed, or returned